SECURITY.md

```diff
@@ -0,0 +1,22 @@
+<h2>Responsible disclosure policy</h2>
+
+At the IOTA Foundation, we consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
+<ul>
+	<li>E-mail your findings to secops@iota.org. If the report contains highly sensitive information, please consider encrypting your findings using our contact@iota.org (466385BD0B40D9550F93C04746A440CCE5664A64) PGP key.</li>
+</ul>
+Please follow these rules when testing/reporting vulnerabilities:
+<ul>
+	<li>Do not take advantage of the vulnerability you have discovered, for example by downloading more data than is necessary to demonstrate the vulnerability.</li>
+	<li>Do not read, modify or delete data that isn't you own.</li>
+	<li>We ask that you do not to disclosure the problem to third parties until it has been resolved.</li>
+	<li>The scope of the program is limited to technical vulnerabilities in IOTA Foundations's web applications and open source software packages distributed through GitHub, please do not try to test physical security or attempt phishing attacks against our employees, and so on.</li>
+	<li>Out of concern for the availability of our services to all users, please do not attempt to carry out DoS attacks, leverage black hat SEO techniques, spam people, and do other similarly questionable things. We also discourage the use of any vulnerability testing tools that automatically generate significant volumes of traffic.</li>
```

Comment:
`and do other similarly questionable things` -> or any other mischief (follow the golden rule)